### PR TITLE
Move the last instance

### PR DESCRIPTION
**Pull request by abbychau**
https://github.com/pingcap/grpc-rs/issues/304

### CODE REVIEW

File: compiler/src/lib.rs [hunk 1]
```diff
@@ -12,5 +12,5 @@
 // limitations under the License.
 
 pub mod codegen;
-pub mod prost_codegen;
+// pub mod prost_codegen;
```
Comment (ice1000) [inline compiler/src/lib.rs]: I suppose this is committed as a mistake.

Comment (abbychau) [reply compiler/src/lib.rs]: done

**Comment by ice1000** [comment]
Please `cargo fmt`:
https://travis-ci.org/pingcap/grpc-rs/jobs/516031657

**Comment by abbychau** [comment]
Sure. I was a bit paranoid doing `cargo fmt` on other people's project.

I configured my global to an enterprise credential. I am now going to git rebase-amend it.

**Comment by abbychau** [comment]
use https://github.com/pingcap/grpc-rs/pull/309

**Pull request closed without merging**